conceptnet5/vectors/sparse_matrix_builder.py

```diff
@@ -57,6 +57,10 @@ def build_from_conceptnet_table(filename, orig_index=(), self_loops=True):
             index1 = labels.add(replace_numbers(concept1))
             index2 = labels.add(replace_numbers(concept2))
             value = float(value_str)
+
+            if dataset == '/d/morphology':
+                value /= 10
```

Comment:
A comment explaining where 10 comes from would be great.

Reply:
Now that I look at it, this was an after-the-fact hack for the fact that I'd given the morphology edges too high a weight and didn't want to rebuild them. Might as well fix it now that I'm going to rebuild them.

The weight should be 0.01, and as always this is pretty arbitrary. The goal is for subword vectors to not affect term vectors much, unless we know very little about the term; meanwhile, because subwords will _only_ have these low-weight links, terms will affect subwords a lot.